spec/units.yaml

```diff
@@ -172,7 +172,16 @@ Ah:
   allowed-datatypes: ['numeric']
 
 # Duration
-
+ns:
+  definition: Duration measured in nanoseconds
+  unit: nanosecond
+  quantity: duration
+  allowed-datatypes: ['numeric']
+us:
+  definition: Duration measured in mikroseconds
```

Comment:
Is it intentional to call it "mikroseconds" rather than "microseconds", the latter seems to be used more frequently.